source/taskmanager/src/main/java/nl/aerius/taskmanager/PriorityTaskSchedulerFileHandler.java

```diff
@@ -34,20 +38,42 @@
  */
 class PriorityTaskSchedulerFileHandler implements SchedulerFileConfigurationHandler<PriorityTaskQueue, PriorityTaskSchedule> {
 
-  private static final String PREFIX = "priority-task-scheduler.";
+  private static final Logger LOG = LoggerFactory.getLogger(PriorityTaskSchedulerFileHandler.class);
+
+  private static final String FILE_PREFIX = "priority-task-scheduler.";
+  private static final String ENV_PREFIX = "AERIUS_PRIORITY_TASK_SCHEDULER_";
 
   private final Gson gson = new GsonBuilder().setPrettyPrinting().excludeFieldsWithoutExposeAnnotation().create();
 
   @Override
   public PriorityTaskSchedule read(final File file) throws IOException {
+    final PriorityTaskSchedule fileSchedule = readFromFile(file);
+    final PriorityTaskSchedule environmentSchedule = readFromEnvironment(fileSchedule.getWorkerQueueName());
+    if (environmentSchedule != null) {
+      return environmentSchedule;
+    }
+    return fileSchedule;
```

Comment:
You can simplify to:
```suggestion
    return environmentSchedule == null ? fileSchedule : environmentSchedule;
```